clang/test/CodeGenHLSL/GlobalConstructorFunction.hlsl

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: directx-registered-target
```

Comment:
This seems wrong. Emitting LLVM from the Clang frontend should not require anything about the set of targets enabled in LLVM itself.